src/v0/destinations/google_adwords_offline_conversions/networkHandler.js

```diff
@@ -201,14 +196,12 @@ const responseHandler = destinationResponse => {
   // the response from destination is not a success case build an explicit error
   // return status, original destination response, message
   const { response } = destinationResponse;
-  throw new ErrorBuilder()
-    .setStatus(status)
-    .setDestinationResponse(response)
-    .setMessage(
-      `[Google Ads Offline Conversions]:: ${response.error.message} during google_ads_offline_conversions response transformation`
-    )
-    .setAuthErrorCategory(getAuthErrCategory(status))
-    .build();
+  throw new AbortedError(
+    `[Google Ads Offline Conversions]:: ${response.error.message} during google_ads_offline_conversions response transformation`,
```

Comment:
I think in future we can try to move these constants `[Google Ads Offline Conversions]` to a common `labels.ts` or some other file name. WDYT?

Reply:
Yes, there should only be one source of truth. However, the destination names should not end up in the error messages.
These kinds of updates are not picked up with this refactoring.